kuka_kr16_support/urdf/kr16r1610cybertech_macro.xacro

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:macro name="kuka_kr16r1610cybertech" params="prefix">
+    <!-- LINKS -->
+    <!-- base link -->
+    <link name="${prefix}base_link">
+      <inertial>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <mass value="2"/>
+        <inertia ixx="0.01" ixy="0" ixz="0" iyy="0.01" iyz="0" izz="0.01" />
+      </inertial>
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <geometry>
+          <mesh filename="package://kuka_kr16_support/meshes/kr16r1610cybertech/visual/base_link.dae" />
```

Comment:
I'm not intimately familiar with the Cybertech family: does it not share any meshes with the regular KR 16 we already have in this package?

Reply:
No, they are visually quite different with the older gen KR 16-2 we got already